compiler/dalgona/CMakeLists.txt

```diff
@@ -1,10 +1,33 @@
+find_package(PythonInterp 3.8 QUIET)
+find_package(PythonLibs 3.8 QUIET)
```

Comment:
We need to update documents, docker, test environment to include these two packages if not already installed. I hope you to update them :)

Reply:
Python 3.8 was already used by common-artifacts. From CMakeLists.txt in common-artifacts

```
find_package(PythonInterp 3.8 QUIET)
find_package(PythonLibs 3.8 QUIET)

if(NOT ${PYTHONINTERP_FOUND})
  message(STATUS "Build common-artifacts: FALSE (Python3 is missing)")
  return()
endif()

if(${PYTHON_VERSION_MINOR} LESS 8)
  message(STATUS "Build common-artifacts: FALSE (You need to install Python version higher than 3.8)")
  return()
endif()
```

Actually I copy-and-pasted those codes :)

So I guess that the change should be limited to dalgona. I uploaded PR to update README (https://github.com/Samsung/ONE/pull/9744). Docker file for dalgona was already updated (https://github.com/Samsung/ONE/pull/9728). Tests for dalgona will be based on Python 3.8.

Reply:
Ah, thanks! I mis-read the package in common-artifacts :)